xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml

```diff
@@ -937,6 +941,15 @@ require(['jquery'], function($) {
   border-color: transparent;
 }
 
+#admin-page-content .usersorgroupsnames {
+  white-space: nowrap;
+  width: 27%;
```

Comment:
This value seems a bit magic.

Reply:
It's the triple of another magic value found for the `.rights` width (which is `9%`).

Unfortunately I'd like to redo a lot of things here (using a table to style the whole content of the section is deffo a mistake in today's standards) but that would be a huge breaking change, would probably create regressions and ask for more work than what I could put in a BFD issue :)

Reply:
Added a comment to explain it in https://github.com/xwiki/xwiki-platform/pull/2985/commits/a996cbc1df6c643d4704521248c04bcbc346ff0c .